feedback/src/main/resources/org/sakaiproject/feedback.properties

```diff
@@ -68,17 +68,14 @@ no_panels_explanation=To display a panel, you need to enable the property for it
 <li>Technical panel: feedback.show.technical.panel=true</li>\
 <li>Suggestion improvement panel: feedback.show.suggestions.panel=true</li></ul>
 content_title=Problem with content?
-content_explanation=<li>Missing file or document?</li>\
-<li>Broken link or blank page?</li>\
-<li>Permission denied?</li>\
-<li>Access problem?</li>\
+content_explanation=<li>Site unavailable or permission denied?</li>\
+<li>Missing file, broken link or blank page?</li>\
```

Comment:
Please use the oxford (serial) comma:

Missing file, broken link, or blank page?